src/templates/action-button.jsx

```diff
@@ -25,14 +25,13 @@ const ActionButton = React.createClass({
         return (
             <HotkeyContainer shortcuts={hotkeys}>
                 <div className="wpnc__action-link">
-                    <a
-                        href="#"
+                    <div
```

Comment:
Note that if the top level changes to a Button, we'll likely want to update the display from `inline-block` to `block`